pkg/config/app_config.go

```diff
@@ -81,21 +81,28 @@ func NewAppConfig(name, version, commit, date string, buildSource string, debugg
 	return appConfig, nil
 }
 
-func ConfigDir() string {
+func SelectDefaultConfiguration() string {
```

Comment:
What are your thoughts on renaming this to `ConfigDir` and then having the existing ConfigDir become a private method called `configDirForVendor`

Reply:
👍 then the change in main.go can be reverted to the old state